projects/Rockchip/devices/OdroidGoAdvance/packages/u-boot/package.mk

```diff
@@ -0,0 +1,83 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (C) 2009-2016 Stephan Raue (stephan@openelec.tv)
+# Copyright (C) 2017-present Team LibreELEC (https://libreelec.tv)
+
+PKG_NAME="u-boot"
+PKG_ARCH="arm aarch64"
+PKG_LICENSE="GPL"
+PKG_SITE="https://www.denx.de/wiki/U-Boot"
+PKG_DEPENDS_TARGET="toolchain swig:host"
+PKG_LONGDESC="Das U-Boot is a cross-platform bootloader for embedded systems."
+
+PKG_IS_KERNEL_PKG="yes"
+PKG_STAMP="$UBOOT_SYSTEM"
+
+[ -n "$ATF_PLATFORM" ] && PKG_DEPENDS_TARGET+=" atf"
+
+PKG_NEED_UNPACK="$PROJECT_DIR/$PROJECT/bootloader"
+[ -n "$DEVICE" ] && PKG_NEED_UNPACK+=" $PROJECT_DIR/$PROJECT/devices/$DEVICE/bootloader"
+
+case "$PROJECT" in
+  Rockchip)
+  if [ "$DEVICE" == "OdroidGoAdvance" ]; then
```

Comment:
Why this if/else? Isn't this u-boot OGA specific? Which platforms use the generic rockchip one?

Reply:
fixed